docs/source/data/errors.mdx

```diff
@@ -40,16 +37,26 @@ Here's an example error response caused by misspelling the `__typename` field in
 
 </ExpansionPanel>
 
-To help with debugging, Apollo Server defines error subclasses that represent different types of errors that can occur while handling a GraphQL operation (such as `SyntaxError` and `ValidationError`). These subclasses each return a different [error code](#error-codes), which enables requesting clients to respond differently to different error types.
+To help with debugging, Apollo Server provides an `ApolloServerErrorCode` enum, which you can use to check if your error is one of the [different types produced by Apollo Server](#error-codes).
 
-These built-in error subclasses inherit from the generic `ApolloError` class, and they're all defined in [the `apollo-server-errors` package](https://github.com/apollographql/apollo-server/blob/main/packages/apollo-server-errors/src/index.ts). You can also [create your own custom errors and codes](#custom-errors).
+If Apollo Server recognizes your error type you can write custom logic to handle that error, like so:
+
+```ts
+import { ApolloServerErrorCode } from '@apollo/server/errors';
+
+if (error.extensions?.code === ApolloServerErrorCode.GRAPHQL_PARSE_FAILED) {
+  // respond to the syntax error
+}
+```
+
+Apollo Server's variety of error codes enables requesting clients to respond differently to different error types. You can also [create your own custom errors and codes](#custom-errors).
 
 ## Error codes
```

Comment:
This section should probably be called "Built-in error codes" or something

Reply:
I like that! Renaming it :)